### ISSUE TITLE
PrefMatch record for non-friend

### ISSUE THREAD

**Issue by jacksenechal** [PeggApp/pegg#39]
I've got an error arising from the fact that this PrefMatch showed up: aAeoHfRXMp. My user (7R64DIa4l4) is not friends with her user (IYdiSw495n) so I get an undefined friend showing up as a match in ParseBackend.coffee:38 in the getTopMatches() method.

The error is "Uncaught TypeError: Cannot read property 'get' of undefined"

**Comment by simple10** [comment]
@jacksenechal do you get the error in tests or just when using the app? I assume just when using the app if it's specific to your user ID.

**Comment by jacksenechal** [comment]
Yeah, exactly, just when using the app. Wouldn't show up in the current
tests since it's an artifact of something else creating bad data.
